python/lsst/ap/association/association.py

```diff
@@ -167,7 +167,7 @@ def run(self,
             diaObjects,
             mergedDiaSourceHistory,
             matchResult.associated_dia_object_ids,
-            filterName)
+            [filterName])
```

Comment:
One question here: how are the filter name(s) passed? They're not already a list, are they? If I look at e.g. python/lsst/pipe/tasks/drpDiaCalculationPipe.py you have 

self.config.filterNames

passed, so it's read from the config. In the actual config is it something like "u","g","r",... as opposed to ["u", "g", ...] my only concern was that you might unexpectedly get a list of lists somehow.

Reply:
filterName here comes from the diaSource catalog itself and will always be a string value here. If you look a few lines above this, it is where filterName is pulled. The pipeline here will only ever be run over one filter as it is designed to run in AP that is one ccdVisit at time. This one runs in the DiaPipeTask pipeline task vs the new wrapper.